store/tikv/test_util.go

```diff
@@ -32,7 +32,7 @@ func NewTestTiKVStore(client Client, pdClient pd.Client, clientHijack func(Clien
 	}
 
 	// Make sure the uuid is unique.
-	uid := uuid.NewV4().String()
+	uid := uuid.New().String()
```

Comment:
irrelevant change?

Reply:
no, twinj/uuid's dependence is too old, upgrade grpc will make go mod conflict, we cannot modify twinj/uuid, so replace it.

this too method generate uuid v4